src/main/kotlin/com/hedvig/underwriter/graphql/Mutation.kt

```diff
@@ -37,6 +39,9 @@ class Mutation @Autowired constructor(
 ) : GraphQLMutationResolver {
 
     fun createQuote(createQuoteInput: CreateQuoteInput, env: DataFetchingEnvironment): CreateQuoteResult {
+
+        logger.info("Create quote: ${createQuoteInput.toNonPiiString()}")
```

Comment:
We use the word "masked" sometimes, maybe you prefer it 🤷 
```suggestion
        logger.info("Create quote: ${createQuoteInput.toMaskedString()}")
```

Reply:
Thanks for good comments!

I will change `toMaskedString()` and to `@Masked`.

Reply:
Fixed in new commit.